src/Symfony/Component/Security/Core/Authorization/Voter/AccessTrait.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter;
+
+/**
+ * @author Dany Maillard <danymaillard93b@gmail.com>
+ */
+trait AccessTrait
+{
+    /** @var int One of the VoterInterface::ACCESS_* constants */
+    protected $access;
```

Comment:
why protected ? This does not make sense as it forces classes using that trait to expose that field to their child classes (while we even have a getter)